.travis.yml

```diff
@@ -19,7 +19,7 @@ before_install:
   - openssl aes-256-cbc -K "$encrypted_5ebd3ff04788_key" -iv "$encrypted_5ebd3ff04788_iv" -in src/bin/travis/resources/jesConf.tar.enc -out jesConf.tar -d || true
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=4648da44fc1695d5306e9b4f05e96fa5a5cadb8f
```

Comment:
I though we just said at standup we wanted the commit hash in here ? xD
I'm asking for the release WDL, do people want a hash or a branch ?